backend/app/adapter/sqldb/user_short_link_integration_test.go

```diff
@@ -114,6 +114,128 @@ func TestListShortLinkSql_FindAliasesByUser(t *testing.T) {
 	}
 }
 
+func TestListShortLinkSql_IsAliasOwnedByUser(t *testing.T) {
+	now := mustParseTime(t, "2019-05-01T08:02:16Z")
+	user := entity.User{
+		ID:             "test",
+		Name:           "mockedUser",
+		Email:          "test@example.com",
+		LastSignedInAt: &now,
+		CreatedAt:      &now,
+		UpdatedAt:      &now,
+	}
+
+	testCases := []struct {
+		name               string
+		userTableRows      []userTableRow
+		shortLinkTableRows []shortLinkTableRow
+		relationTableRows  []userShortLinkTableRow
+		alias              string
+		user               entity.User
+		isFound            bool
```

Comment:
```suggestion
		expectIsFound            bool
```

Reply:
Let's add expect for the expected result so that it's intuitive for readers to figure out the purpose of the test